app/database_etl/airtable_records_handler/airtable_records_formatter.py

```diff
@@ -129,6 +129,11 @@ def add_test_adjustments(df: pd.DataFrame) -> pd.DataFrame:
                   'denominator_value', 'serum_pos_prevalence']
     diff[float_cols] = diff[float_cols].astype(float)
 
+    # Round float columns to a consistent number of decimal places to ensure consistent float comparisons
+    # Defines number of decimal places to round each column to
+    decimals = pd.Series([5]*len(float_cols), index=float_cols)
```

Comment:
gotta admit I had to look up what this was doing! makes sense on lookup.

an alternative way to factor this would be something like
diff[float_cols] = diff[float_cols].round(5)

Reply:
think this is cleaner, will refactor